### PR TITLE
Rename organisation, related to #146

### PR DESCRIPTION
**Pull request by gbraad**
Related to #146 

Compile succeeds with `make`.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -37,8 +37,8 @@ endif
 PYTHON := $(shell command -v python || echo "docker run --rm -it -v $(shell pwd):/minishift -w /minishift python python")
 BUILD_OS := $(shell uname -s)
 
-MINIKUBE_LDFLAGS := -X github.com/jimmidyson/minishift/pkg/version.version=$(VERSION) \
-	-X github.com/jimmidyson/minishift/pkg/version.openshiftVersion=$(OPENSHIFT_VERSION) -s -w -extldflags '-static'
+MINIKUBE_LDFLAGS := -X github.com/MiniShift/minishift/pkg/version.version=$(VERSION) \
```
Comment (jimmidyson) [inline Makefile]: Change to `$(REPOPATH)/pkg/version.version`

File: Makefile [hunk 2]
```diff
@@ -37,8 +37,8 @@ endif
...
+	-X github.com/MiniShift/minishift/pkg/version.openshiftVersion=$(OPENSHIFT_VERSION) -s -w -extldflags '-static'
```
Comment (jimmidyson) [inline Makefile]: Note that I think we will be able to remove this once #141 is worked through.

Comment (hferentschik) [inline Makefile]: +1

**Comment by jimmidyson** [comment]
Closing & reopening to try to trigger travis.

**Comment by gbraad** [comment]
@jimmidyson no worries. However, I did notice that the organisation name is using Pascal Casing, while the organisation for OpenShift uses all lowercase... :-s

**Comment by jimmidyson** [comment]
@gbraad OK let's change org name to minishift - I'm happy with that, but you'll need to update thi PR again if you don't mind.

@hferentschik Happy for me to change org name to lowercase?

**Comment by hferentschik** [comment]
>  Happy for me to change org name to lowercase?

+1. Seems to be the recommended approach anyways. Was just about to suggest the same thing

**Comment by gbraad** [comment]
Will fix this later today

**Comment by jimmidyson** [comment]
Done. Although there's a github redirect for it, please change your remote address & you'll need to move dirs accordingly in GOPATH.

**Comment by hferentschik** [comment]
> Will fix this later today

Thanks :-)

**Comment by gbraad** [comment]
@jimmidyson check. Will also update the CI badges ASAP. (Currently on the subway)

**Comment by LalatenduMohanty** [comment]
works for me 
```
$ make
mkdir -p /Users/lmohanty/code/gocode/src/github.com/minishift/minishift/_gopath/src/github.com/minishift
ln -s -f /Users/lmohanty/code/gocode/src/github.com/minishift/minishift /Users/lmohanty/code/gocode/src/github.com/minishift/minishift/_gopath/src/github.com/minishift
mkdir out 2>/dev/null || true
cd /Users/lmohanty/code/gocode/src/github.com/minishift/minishift/_gopath/src/github.com/minishift/minishift && go run hack/get_openshift.go v1.3.1
Downloading OpenShift v1.3.1
 59.49 MB / 59.49 MB [======================================================================================================================================================================] 100.00% 0sDownloading OpenShift v1.3.1 checksums
 0 B / 947 B [---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------]   0.00%
 947 B / 947 B [============================================================================================================================================================================] 100.00% 0s
GOBIN=/Users/lmohanty/code/gocode/src/github.com/minishift/minishift/_gopath/bin go get github.com/jteeuwen/go-bindata/...
/Users/lmohanty/code/gocode/src/github.com/minishift/minishift/_gopath/bin/go-bindata -nomemcopy -o pkg/minikube/cluster/assets.go -pkg cluster ./out/openshift
CGO_ENABLED=0 GOARCH=amd64 GOOS=darwin go build --installsuffix cgo -ldflags="-X github.com/minishift/minishift/pkg/version.version=0.9.0 -X github.com/minishift/minishift/pkg/version.openshiftVersion=v1.3.1 -s -w -extldflags '-static'" -o ./out/minishift-darwin-amd64 ./cmd/minikube
cp ./out/minishift-darwin-amd64 /Users/lmohanty/code/gocode/bin/minishift
```

**Comment by jimmidyson** [comment]
Thanks @gbraad. Merging.

**Comment by hferentschik** [comment]
Applied to _cluster_up_ branch as well

**Pull request merged**